src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -326,7 +326,36 @@ class AppsController(
 
   private def killTasks(appId: PathId)(implicit identity: Identity): Route = ???
 
-  private def killTask(appId: PathId, taskId: TaskId)(implicit identity: Identity): Route = ???
+  private def killTask(appId: PathId, taskId: TaskId)(implicit identity: Identity): Route = {
```

Comment:
This looks incredibly similar to `killTasks` from #5679. The only difference seems to be `findToKill`. What do you think about consolidating these two diffs? You could make one branch the base of the other.